cli/cmd/predict.go

```diff
@@ -109,8 +110,8 @@ var predictCmd = &cobra.Command{
 		}
 
 		for _, prediction := range predictResponse.Predictions {
-			if prediction.Prediction == nil {
-				prettyResp, err := json.Pretty(prediction.Response)
+			if isExternalModel {
```

Comment:
nit: you can probably do `if resourcesRes.Context.APIs[apiName].IsServingExternalModel()`, or set `api = resourcesRes.Context.APIs[apiName]` above and do `if api.IsServingExternalModel()` here.